axum/src/lib.rs

```diff
@@ -463,7 +463,7 @@ pub use self::form::Form;
 pub use axum_core::{BoxError, Error, RequestExt, RequestPartsExt};
 
 #[cfg(feature = "macros")]
-pub use axum_macros::debug_handler;
+pub use axum_macros::{debug_handler, debug_middleware};
```

Comment:
This reexport should probably also go to the `axum` changelog.

Reply:
https://github.com/tokio-rs/axum/pull/2725/commits/db0b09ec3622ad61f7cd20d4829f18fdb231e8e4